aries-site/src/pages/templates/forms/managing-child-objects.mdx

```diff
@@ -0,0 +1,74 @@
+import { Example } from '../../../layouts';
+
+## When to use
+
+The managing child objects pattern may be used when creating, updating, or removing 
+child objects is part of a larger flow where a parent resource is being created or 
+configured. In these circumstances, the additional configuration inputs can crowd screens 
+and lead to quite lengthy forms.
+
+This pattern improves productivity by focusing a user's attention solely on the current object 
+of interest, and uses collapsible objects to show or hide detail as needed. The 
+intent is that collapsible objects reduce noise, clutter, and the resulting cognitive overload 
+by hiding extraneous detail &mdash; and an easy means to view detail when needed.
+
+## Anatomy
+
+The following illustrates the elements composing a child object in its collapsed and expanded 
+states. View the [use case examples](#use-case-examples) to see how child objects behave and are presented in a form 
+context.
+
+### Collapsed state
+
+[Diagram to be inserted]
+
+| Label | Region         | Purpose                                                    | Required? | Notes |
```

Comment:
```suggestion
| Label | Region         | Purpose                                                    | Required | Notes |
```
It feels a little strange to have the question mark in the column header

Reply:
Agree. I also don't really like the repetitive "required." I was aligning with what has been done on the Layer work, but what if we tried something like:
- column header reads "Required"
- values read either "true" or "<Checkmark a11yTitle="true" />," "optional," or blank
?

cc: @taysea

Reply:
I'm open to that as well, I think I'd tried the true/false approach locally in layers but then found myself scanning back up to the column header to see what "true" / "false" was referring too.

Totally fine giving that a go though if it feels less repetitive.

Reply:
done